elastic/kibana
# chore(rca): notes management without investigation store

Resolves ?

## Summary

This PR removes the usage of the original investigation hook & store to manage the notes. It also uses the real user of the request to populate the createdBy field of the investigation and note.
We refresh the list of notes on creation/deletion (technically, react should not re-render the existing notes, but need to double check), and refresh automatically every ~60~ 10 seconds (should we do it more often)? 
Finally, user can only delete their own notes at the moment.


## Testing

1. Create an investigation (either from the API, or from an alert)
2. Add a note
3. Connect with another user created for the occasion, and go to the investigation. 
4. Add a note with the other user
5. Notes should be sync within 10s

## Screenshot

![image](https://github.com/user-attachments/assets/922d5ab1-b78c-4a6a-b89f-7d86ea7dad76)
